plugins/modules/vmware_guest_instant_clone.py

```diff
@@ -0,0 +1,420 @@
+#!/usr/bin/python
```

Comment:
Add an entry for this module here - https://github.com/ansible-collections/community.vmware/blob/f6ed64f139addcabde07786b91478f3cacfcf258/meta/runtime.yml#L6